### ISSUE TITLE
My alt+left and alt+right has been hijacked!

### ISSUE THREAD

**Issue by massimonewsuk** [ow--/vscode-subword-navigation#30]
So I found these in the Default Keybindings:
```
{ "key": "alt+left",              "command": "subwordNavigation.cursorSubwordLeft",
                                     "when": "editorTextFocus" },
{ "key": "alt+right",             "command": "subwordNavigation.cursorSubwordRight",
                                     "when": "editorTextFocus" },
```

They are normally mapped to:
```
{ "key": "alt+left",              "command": "workbench.action.navigateBack" },
{ "key": "alt+right",             "command": "workbench.action.navigateForward" },
```

This never used to be like this so I don't know why it's happened. I'm using VS Code insiders build.

**Comment by ow--** [comment]
Yeah, as the commit says it was to expose the bindings in the new key binding interface. I chose this as the lesser of two evils and tried to follow platform standards. The thought process can be followed in #27. But as @inosik found, it might work without defaults if I change the activation trigger so I may revisit this.

For now just remove or change the bindings to what you prefer. You can right click a command to show all with the same keybind or just search for a key combination manually.
